plugins/modules/win_get_url.py

```diff
@@ -67,6 +67,12 @@
   url_timeout:
     aliases:
     - timeout
+  set_zone:
+    description:
+      - Markes the file as a file downloaded from internet zone for security purpose
+        using Zone.Identifier stream.
+    type: bool
+    default: no
```

Comment:
Will need a `version_added: 2.5.0` to document when the option was added. I also believe our linting wants boolean values to be `false` and not `no`.